server/app/services/cloud/rpc/server_api.rb

```diff
@@ -55,7 +55,7 @@ def put(user_id, path, options)
 
       def require_user(user_id)
         @user = User.find_by(external_id: user_id)
-        raise RpcServer::Error.new(403, 'Forbidden') unless @user
+        raise RpcServer::Error.new(403, 'Cloud user not found. Please try re-login to master with CLI: kontena master login --force') unless @user
```

Comment:
Yes you are right. Good error message but in wrong place. I still want to tweak error message sent by master a bit, but I'll try to make it more general.